lib/swoosh/adapters/mua.ex

```diff
@@ -1,91 +1,104 @@
-  > [Mail](https://github.com/DockYard/elixir-mail), and
-  > [Castore](https://github.com/elixir-mint/castore) libraries,
-  > add them to your mix.exs file.
-
-  ## Example
+  > This adapter relies on the [Mua](https://github.com/ruslandoga/mua) and
+  > [Mail](https://github.com/DockYard/elixir-mail) libraries.
+  > Ensure they are added to your mix.exs file:
 
       # mix.exs
       def deps do
         [
          {:swoosh, "~> 1.3"},
          {:mua, "~> 0.2.0"},
          {:mail, "~> 0.3.0"},
-         {:castore, "~> 1.0"}
+         # if on OTP version below 25
+         # {:castore, "~> 1.0"}
         ]
       end
 
-      # config/config.exs for sending email directly
+  ## Configuration
+
+  For direct email sending:
+
+      # config/config.exs
       config :sample, Sample.Mailer,
         adapter: Swoosh.Adapters.Mua
 
-      # config/config.exs for sending email via a relay
+  For sending emails via a relay:
+
+      # config/config.exs
       config :sample, Sample.Mailer,
         adapter: Swoosh.Adapters.Mua,
         relay: "smtp.matrix.com",
-        port: 1025,
+        port: 587,
         auth: [username: "neo", password: "one"]
 
+  Define your mailer module:
+
       # lib/sample/mailer.ex
       defmodule Sample.Mailer do
         use Swoosh.Mailer, otp_app: :sample
       end
 
   For supported configuration options, please see [`option()`](#t:option/0)
 
-  ## Sending email directly
+  ## Sending Email Directly
 
-  When `relay` option is omitted, this adapter will send email directly to
-  the receivers' host. All receivers must be on the same host, otherwise
-  `Swoosh.Adapters.Mua.MultihostError` is raised.
+  When the relay option is omitted, the adapter sends emails directly to the recipients' mail servers.
+  All recipients must be on the same host; otherwise, a `Swoosh.Adapters.Mua.MultihostError` is raised.
 
-  In this configuration, you need to ensure that your application can make
-  outgoing connections to port 25 and that your sender domain has appropriate
-  DNS records set, e.g. SPF or DKIM.
+  Ensure your application can make outgoing connections to port 25 and
+  that your sender domain has appropriate DNS records (e.g. SPF, DKIM).
 
-  > #### Short-lived connections {: .warning}
+  > #### Short-lived Connections {: .warning}
   >
-  > Note that each `deliver` call results in a new connection to the receiver's
-  > email server.
+  > Each `deliver/2` call results in a new connection to the recipient's email server.
 
-  ## Sending email via a relay
+  ## Sending Email via a Relay
 
-  When `relay` option is set, this adapter will send email through that relay.
-  The relay would usually require authentication. For example, you can use your own
-  GMail account with an app password.
+  When the relay option is set, emails are sent through the specified relay, typically requiring authentication.
+  For example, you can use your Gmail account with an app password.
```

Comment:
Would it be OK to create a guide like https://gist.github.com/ruslandoga/c94ce526231fb77930132aaeda3fc3c9 and link to it from here or would it be overkill?

Reply:
I'm happy to merge in any docs related improvements. There cannot be too much documentation... or could it? ;)